.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
 
 before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev --no-interaction
+  - composer self-update
+  - composer install --prefer-source --no-interaction
 
 script:
   - mkdir -p build/logs
   - mkdir -p build/cov
-  - php bin/phpunit -c phpunit.xml.dist
+  - phpunit
```

Comment:
@ronanguilloux 

1. Downloading of PHPUnit requires ~10 libs, for every build. Why bother, when this can be handled by travis? There is also  no added value having the most recent version (or is it?). 
2. Usually updated with PHP version update (~1-2 months).
3. I'd vote for removing.